x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/host_isolation_exceptions.ts

```diff
@@ -178,7 +178,18 @@ export default function ({ getService }: FtrProviderContext) {
             .expect(anErrorMessageWith(/invalid ip/));
         });
 
-        it(`should error on [${apiCall.method}] if more than one OS is set`, async () => {
+        it(`should work on [${apiCall.method}] and accept all OSs for os_types`, async () => {
```

Comment:
I don't think you should include this test here. It's a success test, which means you are leaving data behind (specially on create) which could then impact a subsequent test.

if wanting to really include this test, then create its own `it()` outside of the current `describe()` block

Reply:
done c444649fa134cc8d569c0ad19ba1b849ce418ba4. Moved the other success test into a new scope as well.

Reply:
alternatively, we have success tests on at the bottom of this describe block already that we can include the os_types in. I'll be putting in a PR today to include data cleanup for the success tests here and in event filters.